dot/network/service.go

```diff
@@ -292,14 +292,11 @@ func (s *Service) Start() error {
 	// this handles all new connections (incoming and outgoing)
 	// it creates a per-protocol mutex for sending outbound handshakes to the peer
 	// connectHandler is a part of libp2p.Notifiee interface implementation and getting called in the very end
-	//after or Incoming or Outgoing node is connected
+	// after or Incoming or Outgoing node is connected.
 	s.host.cm.connectHandler = func(peerID peer.ID) {
 		for _, prtl := range s.notificationsProtocols {
 			prtl.peersData.setMutex(peerID)
 		}
-		// TODO: currently we only have one set so setID is 0, change this once we have more set in peerSet
-		const setID = 0
-		s.host.cm.peerSetHandler.Incoming(setID, peerID)
```

Comment:
We do not need this code anymore?

Reply:
I was testing this part local and had some doubts about necessety of this part although eventually decided to return it back and rather remove it in separate PR with additional tests.